README.md

```diff
@@ -13,3 +13,25 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+Part 1:
+1. Depending on the memory the machine has, or the program acquires, this can slow runtime.
+2. If you have a more recursion based algorithm and a more iterative approach, these will have different runtimes so cannot be generalized.
```

Comment:
No, iteration and recursion are fundamentally equivalent (as we discussed in class).